lib/process-services/src/lib/task-list/components/task-header/task-header.component.ts

```diff
@@ -72,30 +89,59 @@ export class TaskHeaderComponent implements OnChanges, OnInit {
     dateLocale: string;
 
     private currentUserId: number;
+    private readonly onDestroy$ = new Subject<void>();
+    private readonly usersSubject$ = new BehaviorSubject<CardViewSelectItemOption<number>[]>([]);
+    users$ = this.usersSubject$.asObservable();
 
     constructor(
         private peopleProcessService: PeopleProcessService,
         private translationService: TranslationService,
-        private appConfig: AppConfigService
+        private readonly appConfig: AppConfigService,
+        private readonly cardViewUpdateService: CardViewUpdateService
     ) {
         this.dateFormat = this.appConfig.get('dateValues.defaultDateFormat');
         this.dateLocale = this.appConfig.get('dateValues.defaultDateLocale');
     }
 
     ngOnInit() {
-        this.loadCurrentBpmUserId();
-        this.initData();
+        this.peopleProcessService.getCurrentUserInfo().subscribe((res) => {
```

Comment:
You should unsubscribe this listener at some point

Reply:
good catch, added, thank you